### PR TITLE
add empty service witch migration and db models

### PR DESCRIPTION
**Pull request by AnnPostnikova**
https://trello.com/c/xMGq88V8

### CODE REVIEW

File: Common.Configurations.targets [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  </PropertyGroup>
+
+  <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
+    <PackageReference Include="MediatR" Version="11.0.0" />
+    <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="11.0.0" />
+    <PackageReference Include="DigitalOffice.Kernel" Version="1.0.5" />
+    <PackageReference Include="DigitalOffice.Kernel.BrokerSupport" Version="1.0.0" />
+    <PackageReference Include="DigitalOffice.Kernel.EFSupport" Version="1.0.0" />
+	<PackageReference Include="DigitalOffice.Kernel.ImageSupport" Version="1.0.3" />
+    <PackageReference Include="DigitalOffice.Models" Version="1.0.7" />
```
Comment (vorobevamv) [inline Common.Configurations.targets]: check/update libraries versions

File: src/LibraryService.DataLayer/LibraryServiceDbContext.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
+﻿using LT.DigitalOffice.LibraryService.DataLayer.Models;
+using Microsoft.EntityFrameworkCore;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.LibraryService.DataLayer
+{
+  public class LibraryServiceDbContext :DbContext
```
Comment (vorobevamv) [inline src/LibraryService.DataLayer/LibraryServiceDbContext.cs]: add space LibraryServiceDbContext : DbContext

File: src/LibraryService.DataLayer/Models/DbBook.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+namespace LT.DigitalOffice.LibraryService.DataLayer.Models
+{
+  public class DbBook
+  {
+    public const string TableName = "Books";
+
+    public Guid Id { get; set; }
+    public string Name { get; set; }
+    public Guid? AuthorId { get; set; }
```
Comment (vorobevamv) [inline src/LibraryService.DataLayer/Models/DbBook.cs]: AuthorId is nullable?

File: src/LibraryService.DataLayer/Models/DbBookFile.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+namespace LT.DigitalOffice.LibraryService.DataLayer.Models
+{
+  public class DbBookFile
+  {
+    public const string TableName = "BooksFiles";
+
+    public Guid Id { get; set; }
+    public Guid BookId { get; set; }
+    public Guid FileId { get; set; }
+    public bool IsActive { get; set; }
```
Comment (vorobevamv) [inline src/LibraryService.DataLayer/Models/DbBookFile.cs]: need IsActive here?

File: src/LibraryService.DataLayer/Migrations/20221116150000_InitialCreate.cs [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+      CreateBooksTable(migrationBuilder);
+
+      CreateReviewsTable(migrationBuilder);
+
+      CreateCategoriesTable(migrationBuilder);
+
+      CreateBooksCategoriesTable(migrationBuilder);
+
+      CreateBooksFilesTable(migrationBuilder);
```
Comment (vorobevamv) [inline src/LibraryService.DataLayer/Migrations/20221116150000_InitialCreate.cs]: may be 
CreateTableAuthors
CreateTableBooks...
?

File: src/LibraryService.DataLayer/Models/DbReview.cs [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  {
+    public const string TableName = "Reviews";
+
+    public Guid Id { get; set; }
+    public Guid UserId { get; set; }
+    public Guid BookId { get; set; }
+    public string Content { get; set; }
+    public bool IsActive { get; set; }
+    public DateTime CreatedAtUtc { get; set; }
+    public DateTime? ModifiedAtUtc { get; set; }
```
Comment (vorobevamv) [inline src/LibraryService.DataLayer/Models/DbReview.cs:18]: Can admin modify review? May be add ModifiedBy?

**Review by vorobevamv — APPROVED** [review]

**Review by VLiamin — APPROVED** [review]

File: .editorconfig [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+#prefer fields not to be prefaced with this. or Me. in Visual Basic
+dotnet_style_qualification_for_field = false:suggestion
+#prefer methods not to be prefaced with this. or Me. in Visual Basic
+dotnet_style_qualification_for_method = false:suggestion
+#prefer properties not to be prefaced with this. or Me. in Visual Basic
+dotnet_style_qualification_for_property = false:suggestion
+
+[*.{cs,vb}]
+tab_width=2
+indent_size=2
```
Comment (TIove) [inline .editorconfig]: Add next lines (like in ReactionsService):

[*.{csproj,sln}]
indent_style = space
indent_size=2
tab_width=2

File: src/LibraryService.DataLayer/LibraryServiceDbContext.cs (hunk 2 above)
Comment (TIove) [reply src/LibraryService.DataLayer/LibraryServiceDbContext.cs]: Please, don't close the conversations 'till make changes!

File: src/LibraryService.DataLayer/Models/DbBook.cs (hunk 3 above)
Comment (TIove) [reply src/LibraryService.DataLayer/Models/DbBook.cs]: If you discussed it on the meetings, please, write comments. Like "Discussed on daily meetup"

For now, I think you need to change this.

**Review by maybee16 — APPROVED** [review]

**Review by Vasis3038 — APPROVED** [review]

**Review by TIove — APPROVED** [review]

**Pull request merged**